xkcd/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+// xkcd is an xkcd offline indexing and quering tool.
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/vespian/go-exercises/xkcd/pkg/cmdline"
```

Comment:
/pkg/ is very old school go - just drop it and get
github.com/vespian/go-excersises/xkcd/cmdline